packages/user-operation-controller/package.json

```diff
@@ -74,7 +74,7 @@
     "@metamask/gas-fee-controller": "^19.0.0",
     "@metamask/keyring-controller": "^17.0.0",
     "@metamask/network-controller": "^20.0.0",
-    "@metamask/transaction-controller": "^35.0.0"
+    "@metamask/transaction-controller": "^35.0.1"
```

Comment:
Is this change required? We don't typically bump these unless we need to.

If we need to bump this, it would be a breaking change

Reply:
```suggestion
    "@metamask/transaction-controller": "^35.0.0"
```